app/app/settings.py

```diff
@@ -20,20 +20,21 @@
 # See https://docs.djangoproject.com/en/2.1/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = '@!w0u-pwfozn_f)(w8f)9ou(en7p6v)2!05e0c1&wn(oty2&&s'
+SECRET_KEY = os.environ.get('SECRET_KEY')
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = int(os.environ.get('DEBUG', default=0))
+
+# ALLOWED_HOSTS = ['*']
```

Comment:
If the code I added gives us issues we can switch back to ['*'] as a last resort